activerecord/lib/active_record/persistence.rb

```diff
@@ -622,8 +622,15 @@ def reload(options = nil)
 
     # Saves the record with the updated_at/on attributes set to the current time
     # or the time specified.
-    # Please note that no validation is performed and only the +after_touch+,
-    # +after_commit+ and +after_rollback+ callbacks are executed.
+    #
+    # * No validation is performed.
+    # * Only the +after_touch+, +after_commit+ and +after_rollback+ callbacks
+    #   are executed.
+    # * The dirty-tracking state is cleared for touched attributes, so
+    #   unpersisted changes are no longer tracked by methods like +changed?+,
```

Comment:
The dirty-tracking state is cleared only for touched (= persisted) attributes, so "so unpersisted changes are no longer tracked by methods like ..." part is not true or makes misreading.

Reply:
Can you suggest an alternative wording please? I'm on at least my fourth revision here (all good stuff) so, let's just get it done. Thanks!